### ISSUE TITLE
IdGraphType doesn't trim quotation marks anymore of the value

### ISSUE THREAD

**Issue by JarnoNijboer** [graphql-dotnet/graphql-dotnet#92]
Version: 0.8.0

### Problem
In versions of graphql-dotnet prior to ```0.8.0```, quotation marks where trimmed of the value of an ```IdGraphType``` object. Since ```0.8.0``` this isn't the case anymore and now it's a manual process that must taken in the resolve func of a field.

### Example
When you have a the following query:
```graphql
query getHuman {
  human(id: "1_2") {
    name
  }
}
```
In version ```0.7.x``` when the following code was executed in the resolve func of the human field, the value of the id variable was: ```1_2```
```csharp
var id = context.Argument<string>(@"id");
```
In ```0.8.0``` the value of the id variable will be: ```"1_2"```, and the code will look like
```csharp
var id = context.Argument<string>(@"id").Trim('"');
```

**Comment by JarnoNijboer** [comment]
@joemcbride Thanks for the fast response! 😃

**Comment by joemcbride** [comment]
@JarnoNijboer I figured I would break more than what I realized.  😉  - Published fix in 0.8.1.
